Source/Core/EllipseGeometry.js

```diff
@@ -19,11 +19,9 @@ define([
         './IndexDatatype',
```

Comment:
These file changes shouldn't be here. Make sure your branch is up to date with the `particle-updates` branch by merging that branch into your instead of `master` like normal.